about.php

```diff
@@ -0,0 +1,211 @@
+<?php include'head.php';?>
+
+<div id="pcourses">
+    <span>About us</span>
+</div>
+   <h1>Our Staff</h1>

```

Comment:
If you introduce `<h1>` above for _About us_, we should use `<h2>` here to express the hierarchy of the headlines.